Composer/packages/client/src/CreationFlow/DefineConversation/index.tsx

```diff
@@ -129,8 +127,8 @@ export const DefineConversation: React.FC<DefineConversationProps> = props => {
 
   const handleSubmit = e => {
     e.preventDefault();
-    const errors = validateForm(formData);
-
+    let errors = validateForm(formData);
+    errors = isEmptyName(formData.name);
```

Comment:
This will always overwrite `errors`. Why doesn't `validateForm` check for an empty name?